.travis.yml

```diff
@@ -14,3 +17,7 @@ deploy:
   gem: kontena-plugin-shell
   on:
     tags: true
+after_deploy:
```

Comment:
Fixed already in the more recent commits. Since it has `on tags: true`, it will only deploy when it's a tagged commit.